aws-lambda/massdriver.yaml

```diff
@@ -42,13 +43,14 @@ CloudWatchAlarmMetricFloat: &cloudwatch_alarm_metric_float
     threshold:
       type: number
       title: Threshold
+      description: Percentage in decimal form.
```

Comment:
This won't always be a percentage. If the metric something like utilization it will be percentage. If the metric is storage/memory available/used it will be in bytes, if the metric is a duration it will be seconds.  The threshold is based on the metric being monitored, and the threshold is simply the line that needs to be crossed on the metric to trigger an alarm.